package.json

```diff
@@ -87,21 +87,24 @@
     "@testing-library/react-hooks": "8.0.1",
     "@testing-library/react-native": "11.5.0",
     "@types/react": "^17.0.2",
-    "@types/react-native": "^0.69.2",
+    "@types/react-native": "0.66.4",
+    "@typescript-eslint/eslint-plugin": "^5.44.0",
     "babel-loader": "8.2.2",
     "babel-plugin-module-resolver": "4.1.0",
     "babel-plugin-optional-require": "0.3.1",
     "babel-plugin-react-native-web": "0.17.7",
     "eslint": "^8.23.0",
-    "eslint-config-cozy-app": "^5.1.0",
+    "eslint-config-cozy-app": "^5.2.0",
+    "eslint-plugin-prettier": "4.2.1",
```

Comment:
Ok so this is annoying but for now, the cozy eslint config's plugins aren't reachable by the flagship eslint config. For now, it's best to add the plugins in the app while working towards a solution in `eslint-config-cozy-app`